src/UniOneClient.php

```diff
@@ -21,6 +21,11 @@ final class UniOneClient
      */
     private const VERSION = '0.0';
 
+    /**
+     * The platform name and version.
+     */
+    private string $platform = 'phpsdk.'.self::VERSION;
```

Comment:
To be honest, I don't see any value in moving this into a class property. Can we move it back to the code?